### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by fominmaksim**
Completed dashboard page

### CODE REVIEW

File: src/components/FiveFilmsCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 0.5rem;
+`;
+const StyledSpace = styled(Space)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
```
Comment (AndrewKaenko) [inline src/components/FiveFilmsCarousel.tsx]: These changes can be done with ants

File: src/redux/app/actions.ts [hunk 2]
```diff
@@ -25,3 +27,15 @@ export const loadFighters = () => async (dispatch: Dispatch<AppActionTypes>) =>
   const data = await result.json();
```
Comment (AndrewKaenko) [inline src/redux/app/actions.ts]: Has nothing to do with the project

File: src/redux/app/reducer.ts [hunk 3]
```diff
@@ -11,21 +12,26 @@ export interface Fighter {
 export interface AppStore {
   counter: number;
```
Comment (AndrewKaenko) [inline src/redux/app/reducer.ts]: Has nothing to do with the project

File: src/redux/app/reducer.ts [hunk 4]
```diff
@@ -11,21 +12,26 @@ export interface Fighter {
...
 }
 
 const initialStore: AppStore = {
   counter: 0,
   fighters: [],
+  movies: [],
 };
 
 const app = (store = initialStore, action: AppActionTypes) => {
+  console.log(store);
```
Comment (AndrewKaenko) [inline src/redux/app/reducer.ts]: Unexpected console statement in PR)

File: src/redux/app/reducer.ts [hunk 5]
```diff
@@ -11,21 +12,26 @@ export interface Fighter {
...
   switch (action.type) {
     case AppActions.INC_COUNTER:
       return { ...store, counter: store.counter + action.payload };
     case AppActions.DEC_COUNTER:
       return { ...store, counter: store.counter - action.payload };
     case AppActions.SET_FIGHTERS:
       return { ...store, fighters: action.payload };
```
Comment (AndrewKaenko) [inline src/redux/app/reducer.ts]: Has nothing to do with the project

File: src/redux/app/types.ts [hunk 6]
```diff
@@ -20,5 +22,13 @@ interface AddFightersAction {
...
+interface SetMoviesDataAction {
+  type: AppActions.SET_MOVIES;
+  payload: MostPopularMovie[];
+}
 
-export type AppActionTypes = IncreaseCounterAction | DecreaseCounterAction | AddFightersAction;
+export type AppActionTypes =
+  | IncreaseCounterAction
+  | DecreaseCounterAction
+  | AddFightersAction
```
Comment (AndrewKaenko) [inline src/redux/app/types.ts]: Has nothing to do with the project

File: src/redux/app/types.ts [hunk 7]
```diff
@@ -1,9 +1,11 @@
+import { MostPopularMovie } from '../../services/types';
 import { Fighter } from './reducer';
 
 export enum AppActions {
   INC_COUNTER = 'INC_COUNTER',
   DEC_COUNTER = 'DEC_COUNTER',
   SET_FIGHTERS = 'SET_FIGHTERS',
```
Comment (AndrewKaenko) [inline src/redux/app/types.ts]: Has nothing to do with the project

**Review by AndrewKaenko — CHANGES_REQUESTED** [review]
Corrections required

File: package.json [hunk 8]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
+    "@ant-design/react-slick": "^0.28.4",
```
Comment (alexandrtovmach) [inline package.json]: Do we need that?

File: src/containers/BottomCarousel.tsx [hunk 9]
```diff
@@ -0,0 +1,8 @@
+import FiveFilmsCarousel from '../components/FiveFilmsCarousel';
```
Comment (alexandrtovmach) [inline src/containers/BottomCarousel.tsx]: Why do we need this component? I think it's not doing anything, so can be removed. Instead, you can import FiveFilmsCarousel directly to dashboard.

File: src/containers/MainCarousel.tsx [hunk 10]
```diff
@@ -0,0 +1,7 @@
+import MainCarousel from '../components/MainCarousel';
```
Comment (alexandrtovmach) [inline src/containers/MainCarousel.tsx]: Same, it's redundant component, which is not needed

File: src/containers/RightPanel.tsx [hunk 11]
```diff
@@ -0,0 +1,7 @@
+import SidePanel from '../components/SidePanel';
```
Comment (alexandrtovmach) [inline src/containers/RightPanel.tsx]: Same

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Pull request closed without merging**